jobs/buildhub/lambda_s3_event.py

```diff
@@ -226,15 +229,17 @@
             logger.debug(
                 f"{len(records_to_create)} records to create."
             )
-            for record in records_to_create:
-                # Check that fields values look OK.
-                utils.check_record(record)
-                # Push result to Kinto.
-                kinto_client.create_record(data=record,
-                                           bucket=bucket,
-                                           collection=collection,
-                                           if_not_exists=True)
-                logger.info('Created {}'.format(record['id']))
+            with metrics.timer('s3_event_records_to_create'):
+                for record in records_to_create:
+                    # Check that fields values look OK.
+                    utils.check_record(record)
+                    # Push result to Kinto.
+                    kinto_client.create_record(data=record,
+                                               bucket=bucket,
+                                               collection=collection,
+                                               if_not_exists=True)
+                    logger.info('Created {}'.format(record['id']))
+                    metrics.incr('s3_event_record_created')
```

Comment:
That's out of scope, but I'm surprised we don't use the `batch()` APi here :)

```py
with kinto_client.batch() as batch:
    for record in records_to_create:
        batch.create_record(....)
```

Reply:
What are the chances of there being "many" records to create? If a new file is added in S3, the S3 Event is triggered, which triggers the Lambda function. So almost always there'll only be one record, right?

Reply:
Let's say we have the following S3 events:
1. fr-FR archive, no metadata, no record.
1. en-US archive, no metadata, no record.
1. en-US metadata, scan of locale folders, 2 records.